aggregator/db/migrations/0001.sql

```diff
@@ -1,32 +1,22 @@
 -- +migrate Down
-DROP SCHEMA IF EXISTS aggregator CASCADE;
 
 -- +migrate Up
-CREATE SCHEMA aggregator;
-
-CREATE TABLE IF NOT EXISTS aggregator.batch (
+CREATE TABLE IF NOT EXISTS proof (
 	batch_num BIGINT NOT NULL,
-	batch jsonb NOT NULL,
-	datastream varchar NOT NULL,
-	PRIMARY KEY (batch_num)
-);
-
-CREATE TABLE IF NOT EXISTS aggregator.proof (
-	batch_num BIGINT NOT NULL REFERENCES aggregator.batch (batch_num) ON DELETE CASCADE,
 	batch_num_final BIGINT NOT NULL,
 	proof varchar NULL,
 	proof_id varchar NULL,
 	input_prover varchar NULL,
 	prover varchar NULL,
 	prover_id varchar NULL,
```

Comment:
Nitpick: it is probably more in a fashion of SQL Lite (https://www.sqlite.org/datatype3.html)
```suggestion
	proof TEXT NULL,
	proof_id TEXT NULL,
	input_prover TEXT NULL,
	prover TEXT NULL,
	prover_id TEXT NULL,
```